src/main/java/frc/robot/subsystems/drive/Drivetrain.java

```diff
@@ -165,14 +203,25 @@ public void drive(Vector spdVec, Vector rotVec, boolean altDrive, boolean center
     }
   }
 
+  /**
+   * moves the divetrain based on the given ChassisSpeeds
+   *
+   * @param spds the target speeds of the drivetrain chassis
+   */
+  public void driveChassisSpeeds(ChassisSpeeds spds) {
```

Comment:
How often is this method called?

Reply:
it's called by pathplanner, which needs to take in a function as an argument